app/routes/_index.tsx

```diff
@@ -1,10 +1,12 @@
 import { Card, CardGroup, CardHeader } from "@trussworks/react-uswds";
+import MeetingRoomFAQ from "~/components/MeetingRoomFAQ";
 import { mergeMeta } from "~/lib/merge-meta";
 
 export const meta = mergeMeta(({ parentTitle }) => [{ title: `Meeting Spaces • ${parentTitle}` }]);
 
 export default function Index() {
     return (
+        <>
```

Comment:
Is <> necessary here

Reply:
Yeah, a JSX expression must return a single parent element (we can't have both MeetRoomFAQ and CardGroup be the top level element). That being said, the empty fragment is more or less a placeholder, I'm sure it'll become a div with styling once more components get added to the screen.